### PR TITLE
Implemented dark mode theme for card

### PR DESCRIPTION
**Pull request by Sofiya-kumar**
Screenshot_1701427041

Screenshot_1701427048

Added support for dark theme

### CODE REVIEW

File: Trimble.Modus.Components/Controls/TMCard/TMCard.xaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+             x:Name="card">
+  <Border x:Name="border"
+          BindingContext="{x:Reference card}"
+          Padding="{Binding Padding}">
+    <Border.Shadow>
+      <Shadow Brush="{DynamicResource CardShadowColor}"
+              Radius="{OnPlatform Default=3,Android=15}"
+              Opacity="0.6" />
+    </Border.Shadow>
+    <Border.StrokeShape>
```
Comment (rthanga1) [inline Trimble.Modus.Components/Controls/TMCard/TMCard.xaml:14]: Toggling the 'Selected' property does not change the appearance. It still says in blue color. Can you verify once?

File: DemoApp/DemoApp/Views/CardSamplePage.xaml [hunk 2]
```diff
@@ -124,8 +124,7 @@
           <!-- Footer -->
           <StackLayout BackgroundColor="Transparent"
                        Padding="0,10,0,10">
-            <modus:TMButton Text="Button"
```
Comment (rthanga1) [inline DemoApp/DemoApp/Views/CardSamplePage.xaml:127]: In this page, can you replace the Switch with 'TMSwitch'. Default's switch's UI is bad in iOS

**Review by rthanga1 — CHANGES_REQUESTED** [review]

File: Trimble.Modus.Components/Controls/TMCard/TMCard.xaml (hunk 1 above)
Comment (Sofiya-kumar) [reply Trimble.Modus.Components/Controls/TMCard/TMCard.xaml:14]: Fixed

File: DemoApp/DemoApp/Views/CardSamplePage.xaml (hunk 2 above)
Comment (Sofiya-kumar) [reply DemoApp/DemoApp/Views/CardSamplePage.xaml:127]: Replaced switch modus switch

File: Trimble.Modus.Components/Controls/TMCard/TMCard.xaml (hunk 1 above)
Comment (rthanga1) [reply Trimble.Modus.Components/Controls/TMCard/TMCard.xaml:14]: Maybe you should try VisualStateManager to manage the 'Selected' state.

**Review by ElishaSamPeterPrabhu — APPROVED** [review]

File: Trimble.Modus.Components/Controls/TMCard/TMCard.xaml (hunk 1 above)
Comment (rthanga1) [reply Trimble.Modus.Components/Controls/TMCard/TMCard.xaml:24]: This seems to be fixed now

**Review by rthanga1 — APPROVED** [review]

**Pull request merged**